packages/connect/package.json

```diff
@@ -21,9 +21,9 @@
   ],
   "scripts": {
     "clean:wasm": "make -C crates clean",
-    "build:wasm": "make -C crates all && make -C crates install",
+    "build:wasm": "make -s -C crates all && make -s -C crates install",
```

Comment:
Please don't silence make. Keeping the printout makes debugging the CI easier later.